components/companies-using-kedro/content.tsx

```diff
@@ -0,0 +1,17 @@
+export const content = [
```

Comment:
Love that you moved this to an array. Since it's only this, you could put it in the `companies-using-kedro` file if you wanted to, but your call!